src/api/trajserv/TrajservAPI.test.js

```diff
@@ -0,0 +1,220 @@
25163.250,"y":5949811.500},{"x":925161.125,"y":5949845.500},{"x":925116.062,"y":5950259.500,"a":2019714300,"d":2019714660,"n":"Luzern","dot":0,"put":0,"sid":"8505000:2"},{"x":925161.125,"y":5949845.500},{"x":925163.250,"y":5949811.500},{"x":925161.625,"y":5949780.000},{"x":925154.938,"y":5949720.500},{"x":925129.625,"y":5949614.500},{"x":925104.500,"y":5949542.000},{"x":925084.562,"y":5949455.000},{"x":925076.875,"y":5949433.500},{"x":925071.625,"y":5949422.000,"a":2019714692,"d":0,"ad":60000}],[{"x":924058.875,"y":5949422.000,"a":2019714740,"d":0},{"x":924029.562,"y":5949483.500},{"x":923684.688,"y":5950291.500},{"x":923671.375,"y":5950325.000},{"x":923660.438,"y":5950359.500},{"x":923644.875,"y":5950429.000},{"x":923604.688,"y":5950592.500},{"x":923588.188,"y":5950649.500},{"x":923552.375,"y":5950720.000},{"x":923536.562,"y":5950747.500},{"x":923517.188,"y":5950772.000},{"x":923475.688,"y":5950814.500},{"x":923440.062,"y":5950843.500},{"x":923410.188,"y":5950865.000},{"x":923387.500,"y":5950878.000},{"x":923363.312,"y":5950889.500},{"x":923335.188,"y":5950901.000},{"x":923309.438,"y":5950909.000},{"x":923215.812,"y":5950931.500},{"x":923001.125,"y":5950980.000},{"x":922851.625,"y":5951012.000},{"x":922792.812,"y":5951030.500},{"x":922736.750,"y":5951055.500},{"x":922701.750,"y":5951075.000},{"x":922654.438,"y":5951110.000},{"x":922607.250,"y":5951152.500},{"x":922565.938,"y":5951200.000},{"x":922529.750,"y":5951254.500},{"x":922495.062,"y":5951326.000},{"x":922473.812,"y":5951392.000},{"x":922465.188,"y":5951427.500},{"x":922456.188,"y":5951480.000},{"x":922451.188,"y":5951536.500},{"x":922447.875,"y":5951594.000},{"x":922448.188,"y":5951633.500},{"x":922450.625,"y":5951669.500},{"x":922457.875,"y":5951712.500},{"x":922470.750,"y":5951761.000},{"x":922484.500,"y":5951796.500},{"x":922509.438,"y":5951843.500},{"x":922527.938,"y":5951872.000},{"x":922545.938,"y":5951896.000},{"x":922568.500,"y":5951922.000},{"x":922593.562,"y":5951946.500},{"x":922634.750,"y":5951980.000},{"x":922659.562,"y":5951996.000},{"x":922715.562,"y":5952025.500},{"x":922798.812,"y":5952050.500},{"x":923031.625,"y":5952109.000},{"x":923087.125,"y":5952125.000},{"x":923138.500,"y":5952142.000},{"x":923177.438,"y":5952158.500},{"x":923216.812,"y":5952178.500},{"x":923276.938,"y":5952217.000},{"x":923334.625,"y":5952267.500},{"x":923451.562,"y":5952402.000},{"x":923474.750,"y":5952425.500},{"x":923527.875,"y":5952471.500},{"x":923557.688,"y":5952493.500},{"x":923622.188,"y":5952530.500},{"x":923684.938,"y":5952556.000},{"x":923733.000,"y":5952572.000},{"x":923834.875,"y":5952602.000},{"x":923901.812,"y":5952627.500},{"x":923981.188,"y":5952670.000},{"x":924094.250,"y":5952751.500},{"x":924290.438,"y":5952897.000},{"x":924372.438,"y":5952953.500},{"x":924436.438,"y":5952985.500},{"x":924507.812,"y":5953014.000},{"x":924591.500,"y":5953049.000},{"x":924645.438,"y":5953078.500},{"x":924834.000,"y":5953210.000},{"x":924976.812,"y":5953300.500,"a":2019714971,"d":0,"ad":60000}]]}',
```

Comment:
Same here put this in a well formatted file an import in setuptests.js

Reply:
Moved to a file. Continue in discussion https://github.com/geops/mobility-toolbox-js/pull/12#discussion_r448182290